server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -152,4 +137,61 @@ private String getVideoId(String response) {
     // Only returning the first for now, potential to change if I optimize search results.
     return searchResults.get(0);
   }
+
+  /**
+   * Parses the comment JSON string and retrieves the comments.
+   *
+   * @param response The JSON string to be parsed.
+   * @return An ArrayList with each comment
+   */
+  private ArrayList<String> retrieveComments(String response) {
+    ArrayList<String> commentList = new ArrayList<>();
+
+    // Traversing to the items array
+    JsonElement jElement = JsonParser.parseString(response);
+    JsonObject jObject = jElement.getAsJsonObject();
+    JsonArray itemsArray = jObject.getAsJsonArray("items");
+
+    for (JsonElement el : itemsArray) {
+      // Traversing each item in the array
```

Comment:
Same here, this is just describing mechanics (that the for loop is iterating over items in the array), and doesn't address the intent of the code (which you wouldn't need to in this case).